src/clients_api_coverage.md

```diff
@@ -0,0 +1,71 @@
+# Parsec API coverage within client libraries
+
+The current interface contracts defined for communication between Parsec and its clients is tracking
+the [PSA Cryptography API version
+1.0](https://developer.arm.com/architectures/security-architectures/platform-security-architecture/documentation).
+This page describes the current state of support within clients for the operations, attributes and
+options that are defined in that specification and for Parsec-related functionality (e.g. support
+for various providers).
+
+For an overview of the functionality supported by the service, see the [service API coverage
+page](service_api_coverage.md).
+
+**Note:** Only client libraries that are under our control are represented in the tables below. This
```

Comment:
Maybe "under our control" -> "under the Parsec organization control" or something similar?

Reply:
Is it Parsec organization or ParallaxSecond organisation? Or something else? 😕

Reply:
I'd say Parsec is it is the title of the `parallaxsecond` organization page 😬